web/src/main/java/ca/hendriks/bartender/web/ingredient/IngredientsRestController.java

```diff
@@ -18,14 +18,26 @@ public IngredientsRestController(final IngredientRepository ingredientRepository
     }
 
     @GetMapping
+    @RequestMapping("/ingredients")
     public List<Ingredient> findAllIngredients() {
         return ingredientRepository.findAll();
     }
 
     @PostMapping
     @ResponseStatus(HttpStatus.CREATED)
+    @RequestMapping("/ingredients/add")
     public Ingredient addIngredient(@RequestBody Ingredient ingredient) {
         return ingredientRepository.save(ingredient);
     }
 
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
```

Comment:
It would be incorrect to indicate to the caller that a resource is CREATED (201) when they ask for it to be updated (200). Just remove the annotation